install/kubernetes/helm/istio/charts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -23,7 +23,8 @@ apiVersion: networking.istio.io/v1alpha3
 kind: DestinationRule
 metadata:
   name: "default"
-  namespace: {{ .Release.Namespace }}
+  namespace: istio-config 
+  # {{ .Values.global.configRootNamespace }}
```

Comment:
If the intent is to configure the two namespaces, this is not possible with the 1.2 installer.  WIth 1.3, this should be viable.  At the moment you are limited to one namespace as a result of `tiller` limitations.  Perhaps a newer version fo `helm` permits deploying to multiple namespaces within one chart.  Could you provide `helm version` output?